dockers/docker-ptf/Dockerfile.j2

```diff
@@ -40,7 +40,9 @@ RUN sed --in-place 's/httpredir.debian.org/debian-archive.trafficmanager.net/' /
         wget                \
         cmake               \
         libqt5core5a        \
-        libqt5network5
+        libqt5network5      \
+        iputils-ping
```

Comment:
No. I think Maggie forgot to add the line `RUN dpkg -r inetutils-ping` above line 15.